plugins/main/public/components/common/ribbon/ribbon-item.tsx

```diff
@@ -0,0 +1,146 @@
+import { EuiFlexItem, EuiLoadingSpinner } from '@elastic/eui';
+import React from 'react';
+import { WAZUH_AGENTS_OS_TYPE } from '../../../../common/constants';
+import { AgentStatus } from '../../agents/agent-status';
+import { Agent } from '../../endpoints-summary/types';
+import { WzStat } from '../../wz-stat';
+import { GroupTruncate } from '../util/agent-group-truncate';
+import WzTextWithTooltipIfTruncated from '../wz-text-with-tooltip-if-truncated';
+import './ribbon-item.scss';
+
+const FONT_SIZE = 12;
+
+export enum RibbonItemLabel {
+  GROUPS = 'Groups',
+  OPERATING_SYSTEM = 'Operating system',
+  STATUS = 'Status',
+}
+
+export type IRibbonItem<LABEL extends string = string, VALUE = any> = {
+  key: React.Key;
+  label: LABEL;
+  value: VALUE;
+  style?: React.CSSProperties;
+  isLoading?: boolean;
+  icon?: React.ReactNode;
+};
+
+const isGroups = (
+  item: IRibbonItem,
+): item is IRibbonItem<RibbonItemLabel.GROUPS, string[]> => {
+  return item.label === RibbonItemLabel.GROUPS;
+};
+
+const isOperatingSystem = (
+  item: IRibbonItem,
+): item is IRibbonItem<RibbonItemLabel.OPERATING_SYSTEM, Agent> => {
+  return item.label === RibbonItemLabel.OPERATING_SYSTEM;
+};
+
+const isStatus = (
+  item: IRibbonItem,
+): item is IRibbonItem<RibbonItemLabel.STATUS, Agent> => {
+  return item.label === RibbonItemLabel.STATUS;
+};
+
+interface RibbonItemProps {
+  item: IRibbonItem;
+}
+
+const WzRibbonItem = (props: RibbonItemProps) => {
+  const { item } = props;
+
+  const elementStyle = { ...(item.style || {}), fontSize: FONT_SIZE };
+  const wzWidth100 = { anchorClassName: 'wz-width-100' };
+  const tooltipProps = item.label === 'Cluster node' ? wzWidth100 : {};
+
+  const getPlatformIcon = (agent?: Agent) => {
+    let icon = '';
+    const { uname, platform } = agent?.os || {};
+
+    if (
+      uname?.toLowerCase().includes(WAZUH_AGENTS_OS_TYPE.LINUX) ||
+      platform?.toLowerCase().includes('ubuntu')
```

Comment:
**question**: why is this neccesary? We used the another logic to know if the host is Linux. If your addition for Ubuntu is required, then this should take into account in other views too.

Reply:
I can change for `sysname`
![image](https://github.com/user-attachments/assets/467ffa36-8713-49b9-b0c1-318c1e5b2f93)